scripts/verify-cmux-tui-hosted.sh

```
@@ -162,46 +162,76 @@ fi
+
+# Keep both ends open so the timed read starts before the watcher publishes its
+# result. The watcher owns the GitHub CLI child and reaps it on every exit path.
+exec 3<> "$watch_result_fifo"
+(
+  watcher_pid=""
+  stop_watcher() {
+    if [[ -n "$watcher_pid" ]]; then
+      kill "$watcher_pid" 2>/dev/null || true
+      wait "$watcher_pid" 2>/dev/null || true
+    fi
+  }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)